### ISSUE TITLE
jeo-contrib geotools

### ISSUE THREAD

**Issue by normanb** [jeo/jeo-contrib#4]
I am using the jeo-geotools wrapper and it works well for reading shapefiles! Is it possible to publish this on a public maven repo?

Thanks!

**Comment by jdeolive** [comment]
Thanks for interest in the project, glad it's working well for you! It hasn't been that recently updated in a while but you can find snapshot builds, including the [contrib modules](https://oss.sonatype.org/content/repositories/snapshots/io/jeo/jeo-geotools/), in this repository:

    https://oss.sonatype.org/content/repositories/snapshots

**Comment by ironjan** [comment]
@jdeolive - are the snapshots still updated?

The [latest commit (not counting today's PR)](https://github.com/jeo/jeo-contrib/commit/15916a780cc51e2bdd8f1b0f851b986204ec06e1) was on 2015-09-21, the [latest snapshot of the changed module](https://oss.sonatype.org/content/repositories/snapshots/io/jeo/jeo-solr/0-SNAPSHOT/) was created on  2015-07-23.

**Comment by jdeolive** [comment]
@ironjan Unfortunately not, I hate to say it but the project is more or less dead at this point.

**Comment by ironjan** [comment]
@jdeolive would you be interested in a few more PRs to get the official repository onto jitpack? (jitpack takes github repos, builds them and provides them via their own maven repo)  

I did this [with my fork](https://github.com/ironjan/jeo) and the remaining changes for the official repository would be 
 * adding contrib as git submodule or combining the repos into one
 * add some text to the documentation (along the lines of "unmaintained, PRs welcome, available via jitpack")

**Comment by jdeolive** [comment]
@ironjan Absolutely! Go ahead and submit the PR's and we'll get them merged. Whether to do submodule or single repo I leave up to you... my thought is that just having a single repo would be simpler, and probably more fitting in this case but I don't have a strong opinion here. 

Also if you're game for it we can add you as a member of this org so you can have direct access to the repos. Just let me know and I'll send the invite.
